### PR TITLE
[Teams] add VNC client tutorial

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

File: products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+Cloudflare can render a Virtual Network Computer (VNC) terminal in your browser without any client software or configuration required. 
+
+Administrators can use Cloudflare Tunnel to connect a VNC host to Cloudflare’s network. Using Cloudflare Access, you can apply Zero Trust policies to determine who can access your VNC server. Cloudflare’s network will then enforce the Zero Trust policies and, when a user is allowed, render the client in the browser.
+
+**🗺️ This walkthrough covers how to:**
+- Install and run a Cloudflare Tunnel on a Linux virtual machine
+- Install and configure VNC on a Linux virtual machine
+- Build a Zero Trust policy to determine who can reach the host
+- Render the VNC server in your browser
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md:18]: Should these sentence have `.` at the end?

File: products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+**⏲️Time to complete:**
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md]: Space between emoticon and `Time`

File: products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+## Before you start
+
+1. [Add a website to Cloudflare](https://support.cloudflare.com/hc/en-us/articles/201720164-Creating-a-Cloudflare-account-and-adding-a-website)
+1. [Enable Cloudflare for Teams on your account](/setup)
+1. [Connect your identity provider to Cloudflare for Teams](/identity/sso-integration)
+
+## Configure VNC on your virtual machine
+
+This section walks through installing a VNC server with TightVNC and the Gnome User Interface. If you already have a VNC server installed, you can skip this step.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md]: Not sure but I feel it should be `This section walks **you**...`

File: products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+    ```
+
+1. Once installed, you can create the VNC server instance with the following command:
+
+    ```bash
+    $ sudo tightvncserver
+    ```
+    ![VNC password](../static/zero-trust-security/vnc-client-in-browser/vnc-password.png)
+
+1. Select a password for the VNC server.his password will be used during login for your browser VNC server.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md]: Change `.his password` to `This password`

File: products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+    ```bash
+    $ cloudflared tunnel login
+    ```
+
+1. Create a Tunnel with the command:
+
+    ```bash
+    $ cloudflared tunnel create <NAME>
+    ```
+1. Create a Tunnel configuration file with the command:
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md:127]: Add space above, between the end of ``` and this sentence

File: products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+1. Click **Add an application**.
+
+1. Select **Self-hosted**.
+
+1. Name the application and set the domain to which you would like to expose the VNC server:
+
+    ![Name application](../static/zero-trust-security/vnc-client-in-browser/vnc-domain-application.png)
+
+1. Add a Zero Trust policy. In this example we’re only allowing users with emails ending in `@example.com`.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md]: change `we're` to `we are`?

File: products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+![VNC login screen](../static/zero-trust-security/vnc-client-in-browser/vnc-login-screen.png)
+
+After successful authentication, they may be prompted to enter the VNC server’s password:
+
+![VNC password prompt](../static/zero-trust-security/vnc-client-in-browser/vnc-server-password.png)
+
+
+Then your user will be directed into your VNC server:
+
+![VNC server](../static/zero-trust-security/vnc-client-in-browser/vnc-server.png)
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md:194]: Some of these images are really large and probably don't need to be? Consider using one of the img classes we have to resize them.

File: products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+
+You’ve now successfully deployed a Zero Trust VNC server in the browser. You can define granular access controls across each individual VNC instance. For more information about Cloudflare for Teams, check out our developer guides.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md]: Should `check out our developer guides` have a link to those guides?

**Review by marciocloudflare — APPROVED** [review]

File: products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md (hunk 1 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md:18]: hmm this intro is pretty standardized for Teams tutorials, and we've never had `.`. Let's see what we come up with in terms of `tutorial` strategy before I go on and make any section-wide changes!

File: products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md (hunk 8 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md]: I actually removed the sentence, as the tutorial itself is part of the Teams dev docs

File: products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md (hunk 7 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/tutorials/vnc-client-in-browser.md:194]: the issue with that class is that it justifies all the small images to the left, which is why I've been using it sparingly. I'll see if we can do without some of these screenshots

**Pull request merged**